devlab/tests/generate_load.py

```diff
@@ -557,7 +557,53 @@ def create_routers(self):
         for router in self.config.routers:
             self.neutronclient.create_router(router)
 
-    @clean_if_exists
+    def get_subnet_id(self, name):
+        subs = self.neutronclient.list_subnets()['subnets']
+        for sub in subs:
+            if sub['name'] == name:
+                return sub['id']
+        return None
```

Comment:
Unnecessary `return`. Python returns `None` by default if no return is specified.